keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt

```diff
@@ -10,39 +10,40 @@ import com.netflix.spinnaker.keel.persistence.metamodel.Tables.DELIVERY_ARTIFACT
 import com.netflix.spinnaker.keel.persistence.metamodel.Tables.DELIVERY_ARTIFACT_VERSION
 import org.jooq.DSLContext
 import org.slf4j.LoggerFactory
+import org.springframework.transaction.annotation.Propagation.REQUIRED
+import org.springframework.transaction.annotation.Transactional
 
-class SqlArtifactRepository(
+open class SqlArtifactRepository(
```

Comment:
Still trying to figure out why I had to do this when the `kotlin-spring` plugin is applied and it works in `ResourcePersister`.

Reply:
Seems to be something to do with the fact the annotations are on methods defined in the interface.